src/config/CodeGeneratorConfig.php

```diff
@@ -4,9 +4,9 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CodeGeneratorConfig {
-	
+
 	protected $options;
-	
+
 	public function __construct(array $options = []) {
```

Comment:
If `CodeFileGenerator` will be set to `declareStrictTypes` :  `true`, wouldn't it be good to have `generateScalarTypeHints` and `generateReturnTypeHints` expected to be set to true as well? Is there some kind of validation necessary that checks the consistency of the provided config options? Like a protected 'validate()` method for `CodeGeneratorConfig` that is run at the end of `__construct()` which can be extended by subclasses?